### PR TITLE
fix(nightly): increase nightly delays and timeouts

### PR DESCRIPTION
**Pull request by peusebiu**
change the startup dedupe blackbox test source image with a lighter one

**What type of PR is this?**

**Which issue does this PR fix**:

**What does this PR do / Why do we need it**:

**If an issue # is not available please add repro steps and logs showing the issue**:

**Testing done on this change**:

**Automation added to e2e**:

**Will this break upgrades or downgrades?**

**Does this PR introduce any user-facing change?**:

```release-note

```

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: test/blackbox/restore_s3_blobs.bats [hunk 1]
```diff
@@ -124,10 +125,12 @@ function teardown_file() {
 
     # start with dedupe disabled
     zot_serve ${zot_config_file_nodedupe}
-    wait_zot_reachable 8080
```
Comment (peusebiu) [inline test/blackbox/restore_s3_blobs.bats:127]: no need to wait, just read the log file

File: test/blackbox/pushpull_running_dedupe.bats [hunk 2]
```diff
@@ -119,15 +119,19 @@ function teardown_file() {
     sed -i 's/false/true/g' ${zot_config_file}
 
     zot_serve ${ZOT_PATH} ${zot_config_file}
+
+    # give time to repoDB to parse storage
+    sleep 10
```
Comment (andaaron) [inline test/blackbox/pushpull_running_dedupe.bats:124]: As far as I know zot won't start serving requests until repoDB finishes parsing the storage.

Comment (rchincha) [reply test/blackbox/pushpull_running_dedupe.bats:124]: yes, that should be the preferred way IMO.
Do a curl GET /v2/ check, sleep loop until 200.

File: test/blackbox/restore_s3_blobs.bats (hunk 1 above)
Comment (peusebiu) [reply test/blackbox/restore_s3_blobs.bats:127]: wait_for_string uses tail to catch the log message, if we wait for zot to be reachable startup dedupe may have finished in the meantime. 

I moved the wait_zot_reachable in the second test

File: test/blackbox/pushpull_running_dedupe.bats (hunk 2 above)
Comment (peusebiu) [reply test/blackbox/pushpull_running_dedupe.bats]: then I'll increase the timeouts of curl, it's not enough for this test

File: test/blackbox/restore_s3_blobs.bats (hunk 1 above)
Comment (andaaron) [reply test/blackbox/restore_s3_blobs.bats:127]: Looks good.

**Review by andaaron — APPROVED** [review]

**Pull request merged**